packages/backend/server/migrations/20231010081211_user_features_quotas/migration.sql

```diff
@@ -0,0 +1,31 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `feature` on the `user_feature_gates` table. All the data in the column will be lost.
+  - Added the required column `expires_at` to the `user_feature_gates` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `featureId` to the `user_feature_gates` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "user_feature_gates" DROP COLUMN "feature",
+ADD COLUMN     "activated" BOOLEAN NOT NULL DEFAULT false,
+ADD COLUMN     "expires_at" TIMESTAMPTZ(6) NOT NULL,
+ADD COLUMN     "featureId" VARCHAR NOT NULL;
+
+-- CreateTable
+CREATE TABLE "user_features" (
```

Comment:
change the name to `features`, with the `user_` prefix indicates it is related with users already